lib/rules/jsx-uses-vars.js

```diff
@@ -37,7 +37,7 @@ SOFTWARE.
 module.exports = {
   meta: {
     docs: {
-      description: 'Prevent variables used in JSX to be marked as unused',
+      description: 'prevent variables used in JSX to be marked as unused.', // eslint-disable-line consistent-docs-description
```

Comment:
Why do you disable `consistent-docs-description` rule on this line? Can't you remove the dot?